openassessment/xblock/test/data/update_from_xml_error.json

```diff
@@ -321,6 +321,26 @@
         ]
     },
 
+    "optional_self_assessment": {
```

Comment:
It tests the new enforcement that only staff assessment steps can have a `required` value.

Relevant enforcement: https://github.com/edx/edx-ora2/blob/efischer/enable_staff_grading/openassessment/xblock/xml.py#L575